res/values/strings.xml

```diff
@@ -187,6 +187,7 @@ there are any).</string>
     <string name="changelog_show_full">More…</string>
 
     <string name="widget_name_quickhome">Get Me Home</string>
+    <string name="shortcut_quickhome_long_label">Start search for home.</string>
```

Comment:
I did not think of this and it really looks a bit weird. This text would be shown in launchers with lots of space. But if I think of it "Get Me Home" is more or less all that you need to know. I'll just completely remove that.